Cargo.toml

```diff
@@ -27,7 +27,7 @@ Decomposition and Recomposition, as described in
 Unicode Standard Annex #15.
 """
 
-rust-version = "1.36"
+rust-version = "1.44"
```

Comment:
I'd rather not update MSRV because of deps, rather, we can check in a cargo.lock (or run `cargo update -p tinyvec --precise <someversion>` in the MSRV job, which is my preference so we keep testing latest deps otherwise

Reply:
Good idea, done thanks